package.json

```diff
@@ -64,6 +64,7 @@
     "react-composer": "^5.0.2",
     "react-merge-refs": "^1.1.0",
     "stats.js": "^0.17.0",
+    "three-custom-shader-material": "^3.2.1",
```

Comment:
ThreeJS let’s us ether use a barebones shader material or override existing material using onBeforeCompile.

This dependency is just a fancy wrapper around the latter. It will let us inject custom vertex shader into pre-existing materials. Providing for example, Normal, Phong or Physical shading to the WobbleMaterial.

I could remove the dependency but it would require rewriting code that is already written in the dependency 🤷‍♂️